src/style.css

```diff
@@ -0,0 +1,48 @@
+html {
+  --default-star-image: url(images/star.svg);
+  --active-star-image: url(images/star-active.svg);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: var(--default-star-image) no-repeat center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
+  background-image: var(--active-star-image);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: var(--active-star-image);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: var(--active-star-image);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: var(--active-star-image);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: var(--active-star-image);
+}
```

Comment:
you can combine selectors like 
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
... {
..styles 
}